Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM golang:1.20 as build
+
+WORKDIR /app
+
+COPY go.mod go.sum ./
+RUN go mod download
+
+COPY . .
```

Comment:
Add `config` to `.dockerignore`, along with `*.key` and `*.pem` files. Is `.git` automatically ignored?